doc/man7/EVP_PKEY-FFC.pod

```diff
@@ -0,0 +1,202 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY-FFC - EVP_PKEY FFC key type and algorithm support
```

Comment:
FFC isn't a key type per se.  It's not really anything tangible in this way on it's own, it's a component of the key types DH and DSA, so should rather be presented as such.  I'm not at all sure what to do with this, naming wise.  It possible that we should leave it as `FFC(7)`, but I'm unsure...

Reply:
What if I call it something more like
EVP_PKEY-DH_and_DSA_FFC_params.
And then change the description?

Reply:
I have just changed the description for now.